staging/com.scoreSmart/manifest.tt

```diff
@@ -1,26 +1,31 @@
-class @com.scoreSmart {
+class @com.goal_smart {
   import loader from @org.thingpedia.v2();
   import config from @org.thingpedia.config.none();
+  import config from @org.thingpedia.config.none(api_key=$);
+  
+  
+  entity soccer_team #_[description="Soccer Teams in the World"];
+  entity soccer_league #_[description="Soccer Leagues in the World"];
 
   monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
-                        out Team: String,
-                        out Rank: Number,
-                        out Form: String,
-                        out Points: Number)
-  #_[canonical="get league standings"]
+                        out team: String,
+                        out rank: Number,
+                        out form: String,
+                        out points: Number)
+  #_[result = ["${rank}. ${team} | Form: ${form} | Points: ${points}"]]
+  #_[canonical="the league standings"]
```

Comment:
Omit "the", it will be added later. Also, you'll want to add more paraphrases and synonyms to be more robust to language variations:
```
#_[canonical=["league standings", "league rankings", "soccer rankings"]]
```
(and as many more as you can think of)